juriscraper/opinions/united_states/state/ky.py

```diff
@@ -170,33 +162,19 @@ def _fetch_case_name(self, case_number):
         if len(case_number) != 12:
             return False
 
-        # HTTPS certificate is bad, but hopefully they'll fix it and we can remove the line below
-        self.disable_certificate_verification()
-
         url = (
```

Comment:
Would it be better here to put the GET parameters into a dict and pass them to requests that way?

Reply:
I don't follow what your suggesting.

Reply:
Yeah, I'll restructure